sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_metrics_v1/query.sql

```diff
@@ -1,477 +1,313 @@
-WITH _metrics_ping_distinct_client_count AS (
+WITH baseline_clients AS (
   SELECT
-    DATE(submission_timestamp) AS submission_date,
-    COUNT(DISTINCT client_info.client_id) AS metrics_ping_distinct_client_count
+    ping_date,
+    client_id,
+    channel,
+    country
   FROM
-    firefox_ios.metrics
+    (
```

Comment:
No need for a subquery here. Could use QUALIFY instead. Here's an example:

https://github.com/mozilla/bigquery-etl/blob/3560ac46dae99f48d3c6c2570426d4ff98360ccc/sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_devices_last_seen_v1/query.sql#L86-L87